modules/appeals_api/config/routes.rb

```diff
@@ -5,14 +5,7 @@
   match '/v0/*path', to: 'application#cors_preflight', via: [:options]
 
   namespace :v0, defaults: { format: 'json' } do
-    resources :appeals, only: [:index] do
-      collection do
-        get 'higher_level_reviews/:uuid', to: 'appeals#show_higher_level_review'
-        post 'higher_level_reviews', to: 'appeals#create_higher_level_review'
-        get 'intake_statuses/:intake_id', to: 'appeals#show_intake_status'
-      end
-    end
-
```

Comment:
@camerontesterman I think these were put into v0 by mistake --does that seem right?

Reply:
(or they're no longer relevant)

Reply:
yeah they were added by mistake, should've been removed.

Reply:
thanks!